ocrd/ocrd/processor/base.py

```diff
@@ -107,6 +147,41 @@ def add_metadata(self, pcgts):
                                          value=OCRD_VERSION)])
                     ]))
 
+    def resolve_resource(self, val):
+        """
+        Resolve a resource name to an absolute file path with the algorithm in
+        https://ocr-d.de/en/spec/ocrd_tool#file-parameters
+
+        Args:
+            val (string): resource value to resolve
+        """
+        executable = self.ocrd_tool['executable']
+        if exists(val):
+            return val
+        ret = [cand for cand in list_resource_candidates(executable, val) if exists(cand)]
+        if ret:
+            return ret[0]
+        resmgr = OcrdResourceManager()
+        reslist = resmgr.find_resources(executable, name=val)
+        if not reslist:
+            reslist = resmgr.find_resources(executable, url=val)
+        if not reslist:
+            raise FileNotFoundError("Could not resolve %s resource '%s'" % (executable, val))
+        _, resdict = reslist[0]
+        return str(resmgr.download(
+            executable,
+            url=resdict['url'],
+            name=resdict['name'],
+            path_in_archive=resdict['path_in_archive'],
+            resource_type=resdict['type']
+        ))
+
+    def list_all_resources(self):
+        """
+        List all resources found in the filesystem
+        """
+        return list_all_resources(self.ocrd_tool['executable'])
```

Comment:
Plus append by anything in `pkg_resources.resource_filename` (see comment on `resolve_resource`...)?

Reply:
> Plus append by anything in `pkg_resources.resource_filename` (see comment on `resolve_resource`...)?

We don't want that anymore (https://github.com/OCR-D/spec/commit/fa6e00a269713d8593e77f9b2a3d8f19560e7649)